playwright.config.ts

```diff
@@ -101,10 +101,10 @@ const config: PlaywrightTestConfig = {
   // outputDir: 'test-results/',
 
   /* Run your local dev server before starting the tests */
-  // webServer: {
-  //   command: 'npm run start',
-  //   port: 3000,
-  // },
+  webServer: {
+    command: 'npm run start',
```

Comment:
Nit: You can omit the `run`:

```suggestion
    command: 'npm start',
```

Reply:
Fixed